crates/rpc/rpc/src/eth/api/server.rs

```diff
@@ -245,6 +246,23 @@ where
             .await?)
     }
 
+    /// Handler for: `eth_callMany`
+    async fn call_many(
+        &self,
+        bundle: Bundle,
+        state_context: Option<StateContext>,
+        state_override: Option<StateOverride>,
+    ) -> Result<Vec<EthCallResponse>> {
+        trace!(target: "rpc::eth", ?bundle, ?state_context, ?state_override, "Serving eth_callMany");
+        let res = self
+            .on_blocking_task(|this| async move {
+                this.call_many(bundle, state_context, state_override).await
+            })
+            .await?;
+
+        Ok(res)
```

Comment:
call_many already spawns on the tracing pool and only does non-blocking async work otherwise, so we can call it directly and don't need to spawn a new blocking task here.

Reply:
I'm referencing the `call` function which also spawns on the tracing pool via `transact_call_at -> spawn_with_call_at`. Is that intentional?